test/kpod_diff.bats

```diff
@@ -0,0 +1,28 @@
+#/usr/bin/env bats
+
+load helpers
+
+IMAGE="alpine:latest"
+ROOT="$TESTDIR/crio"
+RUNROOT="$TESTDIR/crio-run"
+KPOD_OPTIONS="--root $ROOT --runroot $RUNROOT --storage-driver vfs"
+
+function teardown() {
+    cleanup_test
+}
+
+@test "test diff of image and parent" {
+    run ${KPOD_BINARY} $KPOD_OPTIONS pull $IMAGE
+    [ "$status" -eq 0 ]
+    runc ${KPOD_BINARY] $KPOD_OPTIONS diff $IMAGE
```

Comment:
I agree.  I wasn't sure how to test containers but that seems like a good option